tests/ethpm/conftest.py

```diff
@@ -40,6 +40,12 @@ def pytest_addoption(parser):
     parser.addoption("--integration", action="store_true", default=False)
 
 
+@pytest.fixture
+def infura_env(monkeypatch):
+    # Please play nice and don't use this key for any shenanigans, thanks!
+    monkeypatch.setenv("WEB3_INFURA_PROJECT_ID", '4f1a358967c7474aae6f8f4a7698aefc')
```

Comment:
I think I'm ok with this, at least as a temporary solution until we figure out a better approach.  You could also use the environment variables and skip+warn when the variable isn't present.